examples/ExamplePageBasic.js

```diff
@@ -46,6 +46,7 @@ class ExamplePageBasic extends Component {
             tools={this.state.tools}
             imageIds={this.state.imageIds}
             style={{ minWidth: '100%', height: '512px', flex: '1' }}
+            isOverlayVisible={false}
```

Comment:
No need to hide the overlay on the basic example, we can remove this.

Reply:
done